azure-pipelines-ci.yml

```diff
@@ -46,4 +51,10 @@ jobs:
   
   steps:
   - template: azure-pipelines/templates/linux-build.yml
+  - template: azure-pipelines/templates/publish.yml
+    parameters:
+      InstallPlatform: Linux-$(BuildPlatform)
   - template: azure-pipelines/templates/test-null-renderer.yml
+    parameters:
+      BuildConfiguration: Debug
```

Comment:
Causes the template to try and execute openblack like it was made by MSVC:

```/__w/_temp/0a54981c-e1b5-4c83-8fa6-18847af35b98.sh: line 3: bin/Debug/openblack: No such file or directory```

Probably just remove?